autopts/pybtp/btp/mesh.py

```diff
@@ -1008,11 +1008,18 @@ def mesh_composition_data_get(net_idx, addr, page):
     stack = get_stack()
     iutctl = get_iut()
 
-    data = bytearray(struct.pack("<HHB", net_idx, addr, page))
-
-    (rsp,) = iutctl.btp_socket.send_wait_rsp(*MESH['composition_data_get'], data)
-
-    (status,) = struct.unpack_from('<B', rsp)
+    req_page = page
+
+    while req_page >= 0:
+        data = bytearray(struct.pack("<HHB", net_idx, addr, req_page))
+        (rsp,) = iutctl.btp_socket.send_wait_rsp(*MESH['composition_data_get'], data)
```

Comment:
Add `Response parameters:` to `btp_mesh_node.txt`?

Reply:
added (Y)